lib/elixir/lib/list.ex

```diff
@@ -390,10 +390,10 @@ defmodule List do
   end
 
   @doc """
-  Wraps the argument in a list.
+  Wraps `term` in a list if this is not list.
```

Comment:
I'm not a native speaker, but this sentence feels strange. "... if it is not already a list." might read better.

Reply:
Yup, good catch. Looks like a typo.